code/modules/multiz/movement.dm

```diff
@@ -227,7 +227,7 @@
 		if(!area2.has_gravity())
 			return
 
-	if(!istype(src, /mob/living/carbon/human))
+	if(!istype(src, /mob/living/carbon/human) && !istype(landing, /turf/space))
```

Comment:
If we're checking those, don't use `istype`. There's a var (`is_hole`) that's true on any turf that can be considered not-solid.